sdk/web-pubsub/web-pubsub-client-protobuf/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "ignorePatterns": ["src/clientProto.js", "src/clientProto.d.ts"],
+  "rules": {
+    "no-return-await": "off",
```

Comment:
do you really want to turn off these rules for the entire package? they're pretty basic and you can always turn them off per-instance instead of globally

Reply:
Copied from another package. Removed.